jp/components/expansion_panel.md

```diff
@@ -292,7 +352,7 @@ export class ExpansionPanelComponent {
     }
 }
 ```
-[**inbuilt suite of animations**]({environment:sassApiUrl}/index.html#animations) の [`slideInLeft`]({environment:sassApiUrl}/index.html#mixin-slide-in-left) と [`slideOutRight`]({environment:sassApiUrl}/index.html#mixin-slide-out-right) アニメーションを使用し、コンポーネントのコンテンツを左側から動的に表示してコンテンツを縮小したときに右側で消えるようにします。既存のパラメーターを使用したい特定のパラメーターでオーバーライドします。
+ビルトイン アニメーション スイートの [`slideInLeft`]({environment:sassApiUrl}/index.html#mixin-slide-in-left) アニメーションと [`slideOutRight`]({environment:sassApiUrl}/index.html#mixin-slide-out-right) アニメーションを使用して、コンテンツを折りたたむとコンポーネントのコンテンツが左側からより動的に表示され、右側から消えるようにします。既存のパラメーターを使用したい特定のパラメーターでオーバーライドします。
```

Comment:
Missing **inbuilt suite of animations** ({environment:sassApiUrl}/index.html#animations) link.